auth0/src/main/java/com/auth0/android/provider/CustomTabsController.java

```diff
@@ -124,6 +124,7 @@ public void run() {
                 Log.d(TAG, "Launching URI. Custom Tabs available: " + available);
 
                 final Intent intent = new CustomTabsIntent.Builder(session.get())
+                        .setShowTitle(true)
```

Comment:
Should this be optional? I am not too keen to force all users for a single person gh issue